File: sqlConnect.php
 <?php
$servername = "localhost";
$username = "jadhas20";
$password = "hase";
$dbname = "jadhas20";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

$sql = "SELECT * FROM students WHERE name='jaden'";
$result = $conn->query($sql);

if ($result->num_rows > 0) {
    // output data of each row
    while($row = $result->fetch_assoc()) {
        echo "Name: " . $row["name"]. " - Age: " . $row["age"]. "Grade Level: -" . $row["gradeLevel"]. "<br>";
    }
} else {
    echo "0 results";
}
$conn->close();
?> 

File: sqlConnect.py
import MySQLdb

db = MySQLdb.connect(host="192.168.0.132", user="user", passwd="password",
db="school")
	#create cursor
cur = db.cursor(MySQLdb.cursors.DictCursor)
#create table requirement
sql = "SELECT * from students"

cur.execute (sql)

rows = cur.fetchall()
for row in rows:
	print(row['name' + " " + str(row['age']) + " " + str(row['gradelevel'])

#disconnect
cur.close()
db.close()
